provider/resources.go

```diff
@@ -110,8 +117,18 @@ func Provider() tfbridge.ProviderInfo {
 				},
 			},
 			"rabbitmq_shovel": {Tok: makeResource(mainMod, "Shovel")},
+			"rabbitmq_operator_policy": {
+				Tok: makeResource(mainMod, "OperatorPolicy"),
+				Docs: &tfbridge.DocInfo{
+					Source: "operator-policy.html.markdown",
+				},
+			},
+		},
+		DataSources: map[string]*tfbridge.DataSourceInfo{
+			"rabbitmq_exchange": {Tok: makeDataSource(mainMod, "getExchange")},
+			"rabbitmq_user":     {Tok: makeDataSource(mainMod, "getUser")},
+			"rabbitmq_vhost":    {Tok: makeDataSource(mainMod, "getVHost")},
```

Comment:
```suggestion
			"rabbitmq_vhost":    {Tok: makeDataSource(mainMod, "getVhost")},
```

^^ that spelling is weird but it's consistent with the way the snake_case maps to CamelCase

Reply:
then again, the corresponding Resource says `VHost` not `Vhost` so maybe it doesn't matter 😭

Reply:
Yeah I was going for consistency with the existing resource